Lib/fontTools/ttLib/tables/C_F_F_.py

```diff
@@ -44,4 +44,4 @@ def toXML(self, writer, otFont, progress=None):
 	def fromXML(self, name, attrs, content, otFont):
 		if not hasattr(self, "cff"):
 			self.cff = cffLib.CFFFontSet()
-		self.cff.fromXML(name, attrs, content)
+		self.cff.fromXML(name, attrs, content, otFont)
```

Comment:
It is there because C_F_F_2.py inherits this method, and needs the otFont argument.